extensions/amp-form/0.1/amp-form.js

```diff
@@ -723,9 +723,12 @@ export class AmpForm {
   handleSsrTemplateFailure_(error) {
     this.setState_(FormState.SUBMIT_ERROR);
     user().error(TAG, 'Form submission failed: %s', error);
+    const errorJsonObject = dict({
+      'error': error['message'],
```

Comment:
I think `error` param's type is wrong. It's actually an `!Error`, not `!JsonObject`, which is part of the confusion.

Then we can just use `error.message` instead of `error['message']` here.

Reply:
fixed.

Reply:
right. fixed.